config/crd/v1/bases/velero.io_schedules.yaml

```diff
@@ -61,6 +61,11 @@ spec:
                 description: Template is the definition of the Backup to be run on
                   the provided schedule
                 properties:
+                  csiSnapshotCreationTimeout:
```

Comment:
Should we add in the description that the default timeout is going to be 10 min ? Also, add in the docs ?

Reply:
Thanks. I will add the default value too.

Reply:
I added some description into site/content/docs/main/examples.md. I didn't find a document is suitable for the parameter. What do you think?